.gitignore

```diff
@@ -1,3 +1,4 @@
 coverage
 dist
 node_modules
+.DS_Store
```

Comment:
Missing empty line at the end of the file.

Reply:
Fixed